api-gateway/src/main/resources/static/scripts/visit-list/visit-list.template.html

```diff
@@ -293,8 +293,8 @@ <h3 style="margin: 0; display: inline-block;">Completed Visits</h3>
         <td class="border">{{v.visitDate | date:'yyyy-MM-ddTHH:mm:ss'}}</td>
         <td class="border" style="white-space: pre-line">{{v.description}}</td>
         <!--        <td style="white-space: pre-line">{{$ctrl.getPractitionerName(v.practitionerId)}}</td>-->
-        <td class="border" style="white-space: pre-line">{{v.practitionerId}}</td>
-        <td class="border" style="white-space: pre-line">{{v.petId}}</td>
+        <td class="border" style="white-space: pre-line">{{v.vetFirstName}} {{v.vetLastName}}</td>
+        <td class="border" style="white-space: pre-line">{{v.petName}}</td>
```

Comment:
Useful UI change Cristian. Makes the front-end more appealing to the user with the Names being present for aggregated data instead of having 36 random characters.